src/app/product-details-page/components/product-description/product-image-preview/zoomit/zoomit.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Input, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-zoomit',
+  templateUrl: './zoomit.component.html',
+  styleUrls: ['./zoomit.component.scss']
+})
+export class ZoomitComponent implements OnInit, OnChanges {
+  // @Input () zsrc: string;
+  zsrc = 'http://www.therxreview.com/wp-content/uploads/2013/05/Reebok-Track-Jacket.png';
```

Comment:
this url should come as Input

Reply:
this line will be deleting after integration and this url will come as Input (it's commented)

Reply:
I can't get the point why did you do so. The integration part should be only cover the case of passing the appropriate data from the parent component, and your component has not to be changed.